app/src/pages/common/settingsPage/settingsPage.jsx

```diff
@@ -106,10 +110,17 @@ export class SettingsPage extends Component {
         component: PropTypes.element.isRequired,
       }),
     ),
+    generalTabExtension: PropTypes.arrayOf(
```

Comment:
Do we need to create a separate extension for a specific tab? Can we use existing extensions (`tabExtensions`)?